tests/test_io_tensor_eager.py

```diff
@@ -278,6 +278,24 @@ def fin():
 
   return args, func, expected
 
+
+@pytest.fixture(name="arrow", scope="module")
+def fixture_arrow(request):
+  import pyarrow as pa # pylint: disable=import-outside-toplevel
+
+  arr1 = pa.array(list(range(100)), pa.int32())
+  arr2 = pa.array(list(range(100)), pa.int64())
+  arr3 = pa.array([x * 1.1 for x in range(100)], pa.float32())
+  table = pa.Table.from_arrays([arr1, arr2, arr3], ['a', 'b', 'c'])
+
+  args = table
+  column = 'b'
```

Comment:
I couldn't figure out how to parametrize the fixture, so this is only testing 1 column. That's probably good enough for now though

Reply:
@BryanCutler There are some ways to parameterize the fixture:
https://stackoverflow.com/questions/42228895/how-to-parametrize-a-pytest-fixture/56871701#56871701

though they are quite complex. Probably not worth the effort for now as the current is good enough.